samples/tab-personal/mvc-csharp/TeamsApp/TeamsApp.ttkproj

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" Sdk="Microsoft.TeamsFx.Sdk">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>42a0237c-30e0-42ae-b6ed-50f318517e77</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="appPackage\color.png" />
+    <None Include="appPackage\manifest.json" />
+    <None Include="appPackage\outline.png" />
```

Comment:
This `ItemGroup` is unnecessary and can be deleted, but it doesn't block usage.

Reply:
Fixed